remote_it/pom.xml

```diff
@@ -77,10 +77,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <!-- org.spark_project.guava -->
-            <groupId>org.apache.spark</groupId>
-            <artifactId>spark-network-common_${scala.binary.version}</artifactId>
-            <version>${spark.version}</version>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```

Comment:
shouldn't we also specify the version?

Reply:
default version is specified in the parent pom.xml:
 <guava.version>14.0.1</guava.version>
or can be changed in the profile